gamecenter/api/views.py

```diff
@@ -15,6 +18,38 @@
 score_schema = ScoreSchema()
 
 
+def handle_api_key(f):
+    @functools.wraps(f)
+    def decorated_function(*args, **kwargs):
+        if "Authorization" in request.headers:
+            api_key = request.headers["Authorization"][7:]  # from the string form "Bearer <api_key>"
```

Comment:
I think it's standard? (small sample size)

[Asana's API](https://asana.com/developers/documentation/getting-started/auth)
[DigitalOcean's API](https://developers.digitalocean.com/documentation/v2/#authentication)

I tried looking up the meaning and got to [this OAuth spec](http://self-issued.info/docs/draft-ietf-oauth-v2-bearer.html). Don't feel like reading that haha. Either way is fine with me.